### PR TITLE
DBZ-4739 Fix Apache Pulsar example

### PR DESCRIPTION
**Pull request by vjuranek**
* Fix `config.properties` to use `schema.include.list`
* Bump debezium version
* Bump Apache Pulsa version
* Fix README to show preformatted text correctly

https://issues.redhat.com/browse/DBZ-4739

### CODE REVIEW

**Comment by vjuranek** [comment]
Should this be removed as well @gunnarmorling ? Same as PR #205 ? Pulsar seem to be [integrated into Debezium](https://debezium.io/documentation/reference/stable/operations/debezium-server.html#_apache_pulsar) server as well.

**Comment by gunnarmorling** [comment]
Hey @vjuranek, we could remove it, yes. OTOH I think it's nice to have at least one example for showing the usage of the embedded engine, so I'd suggest to keep this one, but add a note at the top of the README, saying something like this:

> This example shows how to consume change events programmatically using the Debezium embedded engine mode [link to docs]. If you want to stream change events into Apache Pulsar, take a look at the ready-made sink for Debezium Server [link to docs].

**Comment by vjuranek** [comment]
Makes sense. Readme updated.

File: apache-pulsar/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Streaming Postgres Database Changes to Apache Pulsar Using Debezium
 
-[Debezium](https://debezium.io/) allows to capture and stream change events from multiple databases such as MySQL and Postgres and is mostly used with Apache Kafka as the underlying messaging infrastructure.
-
-Using [Debezium's embedded mode](https://debezium.io/docs/embedded/) it is possible to stream database changes to arbitrary destinations and thus not be limited to Kafka as the only broker.
-This demo shows how to stream changes from a Postgres database to Apache [Pulsar](https://pulsar.incubator.apache.org/).
+This example shows how to consume change events programmatically using the [Debezium's embedded mode](https://debezium.io/docs/embedded/),
+This approach allows to stream database changes to arbitrary destinations.
+The demo shows how to stream changes from a Postgres database to Apache [Pulsar](https://pulsar.apache.org/).
+However, if you want to stream change events into Apache Pulsar, take a look at the ready-made sink for [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html#_apache_pulsar).
```
Comment (gunnarmorling) [inline apache-pulsar/README.md:6]:
```suggestion
This example shows how to consume change events programmatically using the [Debezium's embedded mode](https://debezium.io/documentation/reference/stable/development/engine.html),
This approach allows to stream database changes to arbitrary destinations.
The demo shows how to stream changes from a Postgres database to Apache [Pulsar](https://pulsar.apache.org/).
However, if you want to stream change events into Apache Pulsar in a production scenario, take a look at the ready-made sink for [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html#_apache_pulsar).
```

**Comment by gunnarmorling** [comment]
Applied, thanks a lot. Note I've changed the first docs URL to refer to https://debezium.io/documentation/reference/stable/development/engine.html. the "docs" path is from the legacy docs structure years ago, and while we do have redirects in place for taking the user to the right page, we only should share the "stable" links as above, so as to make sure they keep the highest SEO ranking and show up first in Google searches (also it's nice for the user to save a redirect ofc).

**Pull request merged**